visualization/app/codeCharta/util/urlExtractor.spec.ts

```diff
@@ -72,7 +72,7 @@ describe("urlExtractor", () => {
 			expect(urlExtractor.getFileDataFromFile).toHaveBeenCalledWith({ data: "some data" })
 		})
 
-		it("should return an array of resolved file data", () => {
+		it("should return an array of resolved file data", async () => {
```

Comment:
is `async` even required when we don't use await?
I see the benefit of telling the developer that some asynchronous code execution is happening here, but the keyword shouldn't be required.

Reply:
The behavior is a tad different when using `async` as keyword. This is a lint rule that I personally would keep. It's otherwise difficult to determine that it's indeed an async function.